api/app/main.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import FastAPI, APIRouter
+
+# from .config import settings
```

Comment:
commented on purpose?

Reply:
yep, mainly as an example how to use it :)